### PR TITLE
Fix #19206: Missing cron task for application of monitoring templates

### PR DESCRIPTION
**Pull request by julienbriault**

### CODE REVIEW

File: zabbix/packaging/postinst [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+CRONFILE="/etc/cron.d/zabbix-rudder"
 
 mkdir -p /var/rudder/plugin-resources
-echo "0 * * * * /opt/rudder/bin/zabbix-plugin.py update " > /etc/cron.d/zabbix-rudder
+
+# create cron file
+if [ ! -f "${CRONFILE}"]; then
+  cat > "${CRONFILE}" <<EOF
+0 * * * * /opt/rudder/bin/zabbix-plugin.py update 
+*/2 * * * * root /opt/rudder/bin/centreon-plugin apply-configuration >/dev/null
```
Comment (amousset) [inline zabbix/packaging/postinst:12]: why a line linked to centreon here?

**Comment by julienbriault** [comment]
Correction made. The value "centreon" has been replaced by "zabbix".

File: zabbix/packaging/postinst [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 CONFFILE="/opt/rudder/etc/zabbix.conf"
+CRONFILE="/etc/cron.d/zabbix-rudder"
 
 mkdir -p /var/rudder/plugin-resources
-echo "0 * * * * /opt/rudder/bin/zabbix-plugin.py update " > /etc/cron.d/zabbix-rudder
+
+# create cron file
+if [ ! -f "${CRONFILE}"]; then
+  cat > "${CRONFILE}" <<EOF
+0 * * * * /opt/rudder/bin/zabbix-plugin.py update
```
Comment (amousset) [inline zabbix/packaging/postinst:11]: this line is missing the user to use

**Comment by julienbriault** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PRs should in general be done in the oldest affected branch. In this case it should be 6.1.

**Pull request closed without merging**